numba/dppl/target.py

```diff
@@ -51,6 +51,9 @@ def load_additional_registries(self):
 SPIR_VERSION = (2, 0)
 
 
+LINK_ATOMIC = 111
```

Comment:
What is this?

Reply:
Option was added to dppl target context to identify if we need to link our kernel with any other implementation, e.g. **only kernels** containing atomics should be linked with spirv binary containing the implementation of those atomic operations. The options are passed in form of dictionary where the keys define which object files need to be linked. Here, we are defining that "key" for linking the atomic.

Reply:
Simply beacause I prefer avoiding string whenever I can. I understand it will be hashed and might not make a big difference but I still prefer using integer.